SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -281,7 +276,7 @@ void USpatialActorChannel::RetireEntityIfAuthoritative()
 				NetDriver->ActorSystem->RetireEntity(EntityId, Actor->IsNetStartupActor());
 			}
 		}
-		else if (bCreatedEntity) // We have not gained authority yet
+		else if (bCreatingNewEntity) // We have not gained authority yet
```

Comment:
This one was fun
I think this change is correct, since we were checking for entities that should have had a CreateEntity request in flight. But I am slightly confused about the flows here, so could talk with someone about it.

Reply:
it was not correct